### DISCUSSION TITLE
What is the new alternative for  Daf mobile app

### DISCUSSION THREAD

**Discussion by devrajsinghrawat** [decentralized-identity/veramo#648] [Q&A]
Dear Team,

As you are phasing out the Daf mobile app, what is the client side recommended app to be use to test the functioning at Veramo.

**Comment by jasheal** [comment]
Hi @devrajsinghrawat 

Check out our post about the agent explorer
https://veramo.io/blog/open_sourcing_agent_explorer

Also, the tutorial to set up Veramo mobile 
https://veramo.io/docs/react_native_tutorials/react_native_setup_identifers

We still support mobile on React Native but currently do not have the resources to maintain full iOS / Android examples as more time is spent debugging non Veramo related issues. We are also hoping to support expo soon which should make maintaining an example somewhat easier.
